crate/pyo3/python/scripts/test_kms.py

```diff
@@ -5,226 +5,228 @@
     MasterSecretKey,
     Policy,
     PolicyAxis,
-    PublicKey,
+    MasterPublicKey,
     UserSecretKey,
 )
 from cosmian_kms import KmsClient
 
 
 class TestKMS(unittest.IsolatedAsyncioTestCase):
     async def asyncSetUp(self) -> None:
-        self.client = KmsClient("http://localhost:9998")
+        self.client = KmsClient('http://localhost:9998')
 
         # Create Policy
         self.policy = Policy()
         self.policy.add_axis(
             PolicyAxis(
-                "Security Level",
+                'Security Level',
                 [
-                    ("Protected", False),
-                    ("Confidential", False),
-                    ("Top Secret", False),
+                    ('Protected', False),
+                    ('Confidential', False),
+                    ('Top Secret', False),
                 ],
                 hierarchical=True,
             )
         )
         self.policy.add_axis(
             PolicyAxis(
-                "Department",
-                [("FIN", False), ("MKG", False), ("HR", False)],
+                'Department',
+                [('FIN', False), ('MKG', False), ('HR', False)],
                 hierarchical=False,
             )
         )
 
         # Generate master key pair
         (
-            self.pubkey_uid,
-            self.privkey_uid,
+            self.pub_key_uid,
+            self.priv_key_uid,
         ) = await self.client.create_cover_crypt_master_key_pair(self.policy.to_bytes())
 
     async def test_master_keys(self) -> None:
         # Query public key from KMS
-        pubkey = await self.client.get_object(self.pubkey_uid)
-        self.assertEqual(pubkey.object_type(), "PublicKey")
-        self.assertIsInstance(PublicKey.from_bytes(pubkey.key_block()), PublicKey)
+        pub_key = await self.client.get_object(self.pub_key_uid)
+        self.assertEqual(pub_key.object_type(), 'PublicKey')
```

Comment:
I am not a fan of the single quoted strings.

Reply:
There is a rule in `.pre-commit-config.yaml` named `double-quote-string-fixer` that does the fix everywhere. Should I remove it?

Reply:
This is mostly a question of taste I think. But I would argue that double quoted strings are the "natural" way to write them.

So I would remove it. But if you don't, then don't :)

Reply:
For the time being, I will let it as single quote. And probably a dedicated PR will be better